builds/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,19 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.easyrpg.player"
+    android:installLocation="auto"
     android:versionCode="1"
-    android:versionName="1.0"
-    android:installLocation="auto">
-
+    android:versionName="1.0">
     <!-- OpenGL ES 2.0 -->
-    <uses-feature android:glEsVersion="0x00020000" />
-
-    <!-- Touchscreen support -->
+    <uses-feature android:glEsVersion="0x00020000" /> <!-- Touchscreen support -->
```

Comment:
Looks like a lot of blank lines got stripped here, leading to misaligned comments.

Reply:
This could be formatted by Android Studio IDE itself, maybe is not worth to restore positioning by hand if later gets rewritten again